### ISSUE TITLE
Person post lists - remove text clipping on email addresses

### ISSUE THREAD

**Issue by cjg89** [UCF/UCF-WordPress-Theme#112]
**Describe the bug**

https://github.com/UCF/UCF-WordPress-Theme/blob/221fc73145344bb4761d8fea2a84f8d7f254cc23/src/scss/components/_post-list.scss#L33-L37

`overflow: hidden;`, `text-overflow: ellipsis;`, and `white-space: nowrap;` should be removed/adjusted from the snippet linked above to ensure that users zoomed up to 200% will not have email contents clipped that users at a standard zoom level would be able to see.

**Steps to Reproduce**

n/a

**Expected Behavior**

Text should never be clipped/truncated more when users zoom their browser up to 200%.

**Possible Fix**

Remove the rules noted above; look into `overflow-wrap: break-word` to ensure long email addresses are able to break mid-string and not cause layout problems: https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap

Alternatively, consider replacing email addresses with generic "Email" text with envelope icon to better suit the limited horizontal space.  (Use `.sr-only` text within each link with the person's name or email address to ensure they're unique)

**Your environment:**

n/a

**Additional context**

https://www.w3.org/WAI/WCAG21/Understanding/resize-text
